plugins/callback/logentries.py

```diff
@@ -6,79 +6,80 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-DOCUMENTATION = '''
-    author: Unknown (!UNKNOWN)
-    name: logentries
-    type: notification
-    short_description: Sends events to Logentries
+DOCUMENTATION = r"""
+author: Unknown (!UNKNOWN)
+name: logentries
+type: notification
+short_description: Sends events to Logentries
+description:
+  - This callback plugin will generate JSON objects and send them to Logentries using TCP for auditing/debugging purposes.
+  - Before 2.4, if you wanted to use an ini configuration, the file must be placed in the same directory as this plugin and
+    named C(logentries.ini).
+  - In 2.4 and above you can just put it in the main Ansible configuration file.
```

Comment:
I don't think anyone is still interested in how this can be configured with Ansible 2.3 and earlier :)
```suggestion
```